capstone/src/main/java/com/step902020/capstone/EventController.java

```diff
@@ -52,18 +52,17 @@ public RedirectView saveEvent (
      @RequestParam("requiredFee") Optional<Boolean> requiredFee,
      @RequestParam("event-id") String eventId
     ) throws IOException {
-      Organization organization = organizationRepository.findByEmail(user.getEmail()).get(0);
+      Organization organization = organizationRepository.findByEmail(user.getEmail()).orElse(null);
```

Comment:
See the previous PR for suggestion on simplifying this with a custom repository method name.